index.html

```diff
@@ -0,0 +1,911 @@
+          A, Page 1 of the New York edition with the headline: Space Ripples
+          Reveal Big Bang’s Smoking Gun. <a class="linktext" href="#">Order Reprints</a>|
+          <a class="linktext" href="#">Today’s Paper</a> | <a class="linktext" href="#">Subscribe</a> 
+        </span>
+        <br><br>
+
+         <!-- Comments and Social Media Buttons -->
+
+
+        <div class="head-new">
+          <button class="buttons" >READ 615 COMMENTS</button>
+          <div class="head-new-buttons">
+            <span class="medialogo">
+              <img src="assets/images/facebook.svg" alt="facebook logo" class="media-icon"/>
+            </span>
+  
+            <span class="medialogo">
+              <img src="assets/images/twitter.svg" alt="twitter logo" class="media-icon">
+            </span>
+  
+            <span class="medialogo">
+              <img src="assets/images/email.svg" alt="email logo" class="media-icon">
+            </span>
+  
+            <span class="medialogo">
+              <img src="assets/images/share.svg" alt="share logo" class="media-icon">
+            </span>  
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!-- Aside Section -->
+
+    <div class="asides">
+         
+      <aside class="MostPopular">
+
+        <hr class="asides-hr">
+        <h5 class="bolder-titles"><a href="#">Most Popular</a></h5>
+
+        <hr />
+        <p class="popular-item">
+          Watching South Korean Baseball on TV? Let Us Help
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          15 Children Are Hospitalized With Mysterious Illness Possibly Tied to
+          Covid-19
+        </p>
+
+        <hr />
+        <p class="popular-item">Opinion: The Leader We Wish We All Had</p>
+
+        <hr />
+        <p class="popular-item">
+          In Japan, the ‘Murder Hornet’ Is Both a Lethal Threat and a Tasty Treat
+        </p>
+
+        <hr />
+        <p class="popular-item">Opinion: Trump and His Infallible Advisers</p>
+
+        <hr />
+        <p class="popular-item">
+          For Joanna Gaines, Home Is the Heart of a Food and Design Empire
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          Working From Home, Celebrities Grapple With Revealing Too Much
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          How to Cook and Freeze a Large Piece of Meat, and Eat for Weeks
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          What if You Don’t Want to Go Back to the Office?
+        </p>
+
+        <hr />
+        <p class="popular-item">
+          3 Hospital Workers Gave Out Masks. Weeks Later, They All Were Dead.
+        </p>
+      </aside>
+      <hr class="asides-hr">
+      <h5 class="bolder-titles"><a href="#">More in Space and Astronomy</a></h5>
+      <aside class="SpaceAstronomy rows-asside">
```

Comment:
In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))